Create ui-c8y-1019-24-4-user-message-in-device-list-page-is-incorrect-when-no-devices.md (#2076)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1019-24-4-user-message-in-device-list-page-is-incorrect-when-no-devices.md

---
date: ""

> title: User message in device list page is incorrect when no devices are available in the tenant [REVERT][GRAFT][release/cd]

Author: The Claude AI suggested to use the following title:
```suggestion
title: Revert: Incorrect user message on device list page when no devices are available
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-1214
version: 1019.24.4
---

> User message in device list page is incorrect when no devices are available in the tenant [REVERT][GRAFT][release/cd]

Author: The Claude AI suggested to use the following body:
```suggestion
In a previous change, the user message displayed on the device list page when no devices were available in the tenant was updated. However, this change introduced an incorrect message. To address this, the previous change has been reverted. With this revert, when no devices are available in the tenant, the device list page now correctly displays the appropriate message to inform users about the absence of devices. This ensures a clear and accurate user experience when navigating the device list page in scenarios where no devices are present.
```
